x-pack/test/security_api_integration/tests/session_idle/extension.ts

```diff
@@ -38,22 +40,26 @@ export default function ({ getService }: FtrProviderContext) {
       sessionCookie = parseCookie(response.headers['set-cookie'][0])!;
       return response;
     };
-    const getSessionInfo = async () =>
-      supertestWithoutAuth
+    const getSessionInfo = async () => {
```

Comment:
__question:__ I understand why we might need `es.indices.refresh` inside of `getSessionsCreatedAt` (since it relies on `search`), but not sure we need `refresh` inside `getSessionInfo` and `extendSession` since they both shouldn't use `search` under the hood, or am I missing something?